src/pages/methods/workflow-details/WorkflowWrapper.test.tsx

```diff
@@ -1,3 +1,4 @@
+import { expect } from '@storybook/test';
```

Comment:
I think this might have been added by accident. We normally use Jest's `expect` that is available without needing an import, and this `expect` is overriding it and causing a lot of warnings.

Reply:
Oops. Didn't notice that. I will update it 👍